airgun/entities/host_new.py

```diff
@@ -18,13 +18,21 @@
     ParameterDeleteDialog,
     RemediationView,
 )
+from airgun.views.hostgroup import HostGroupEditView
 from airgun.views.job_invocation import JobInvocationCreateView
 
 global available_param_types
 available_param_types = ['string', 'boolean', 'integer', 'real', 'array', 'hash', 'yaml', 'json']
 
 
 class NewHostEntity(HostEntity):
+    def common_setup(self, entity_name, role_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name, role=role_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        view.overview.details.edit.click()
+        self.browser.switch_to_window(self.browser.window_handles[1])
```

Comment:
you can include below mentioned step as well. try to give the name to function more authentic e.g. navigate_to_host_group_edit_view   

```
return HostGroupEditView(self.browser)
```